frontend/src/containers/LibraryPage/LibraryPage.js

```diff
@@ -90,29 +98,54 @@ class LibraryPage extends React.Component {
   toggleDeletionDialog = () => {
     const { showDeletionDialog } = this.state;
     this.setState({ showDeletionDialog: !showDeletionDialog });
-  }
+  };
 
   handleVideoDeletion = () => {
     const { selectedCards } = this.state;
-    deleteVideos(selectedCards)
+    markDeleted(selectedCards)
```

Comment:
deleteMarked?

Reply:
MarkedForDeletion?

Reply:
Nope. It's not gonna delete, it's gonna mark